HACKING.md

```diff
@@ -49,7 +49,7 @@ Or a rpm package: (from the cloned `touchegg` folder)
 
 ```bash
 $ mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
-$ tar -czvf ~/rpmbuild/SOURCES/touchegg.tar.gz -C .. touchegg
+$ tar -czvf ~/rpmbuild/SOURCES/touchegg.tar.gz -C .. touchegg  --transform 's/^touchegg/touchegg-2.0.10/'
```

Comment:
Why is this transform required? I'd like to avoid it, as it changes between releases.

Reply:
I tried to avoid it too. I agree, it does look ugly. The problems is that the `git clone` command that copr uses build the `src.rpm` straight from the git repo puts all source in a `%{name}-%{version}` subdir. We can avoid this by using other `src.rpm` build methods, for example just creating it locally and uploading it manually, but with this you will loose the automation that can build the package with each commit to the branch.

Reply:
You were right in your other comment, the RPM package is not following the standard name-version. We can avoid typing the version thought:

```suggestion
$ tar -czvf ~/rpmbuild/SOURCES/touchegg.tar.gz -C .. touchegg --transform s/^touchegg/touchegg-$(git describe --tags --abbrev=0)/
```